src/rule_templates/internal-failure@__device_ups__.rule

```diff
@@ -0,0 +1,12 @@
+{
+    "single" : {
+        "rule_name"     :   "internal-failure@__name__",
+        "rule_class"    :   "TRANSLATE_LUA(UPS has an internal failure)",
+        "target"        :   ["status.ups@__name__", "ups.alarm@__name__"],
+        "element"       :   "__name__",
+        "values"        :   [ ],
+        "results"       :   [
+            {"high_critical"  : { "action" : [{ "action": "EMAIL" },{ "action": "SMS" }], "severity" : "CRITICAL", "description" : "{\"key\" : \"TRANSLATE_LUA(UPS $ename$ has an internal failure!)\", \"variable\" : {\"ename\" : \"__ename__\"} }" }} ],
```

Comment:
let's allow the user to set the action he want. So define an empty array { "action" : [], "sevirity" ...

Reply:
Remove the exclamation mark from the label. 
s/internal failure!/internal failure/g
Note : the exclamation mark is used by NUT as a separator character